.github/workflows/ci.yml

```diff
@@ -0,0 +1,111 @@
+name: CI
+
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+
+jobs:
+  rspec:
+    name: Ruby ${{ matrix.ruby }} / ${{ matrix.gemfile || 'Gemfile' }}
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - { ruby: 2.3 }
+        - { ruby: 2.4 }
+        - { ruby: 2.5 }
+        - { ruby: 2.6 }
+        - { ruby: 2.6, gemfile: 'Gemfile.rails_next' }
+
+    services:
+      postgres:
+        image: fixmystreet/postgres:latest
+        env:
+          POSTGRES_PASSWORD: postgres
+        ports:
+        - '5432:5432'
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    env:
+      BUNDLE_GEMFILE: ${{ matrix.gemfile || 'Gemfile' }}
+      DATABASE_URL: postgres://postgres:postgres@localhost:5432/alaveteli_test
+      RAILS_ENV: test
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Install packages
+      env:
+        DEBIAN_FRONTEND: noninteractive
+      run: |
+        sudo apt-get -y install exim4-daemon-light
+        sudo apt-get -y install `cut -d " " -f 1 config/packages.ubuntu-bionic | egrep -v "(^#|wkhtml|bundler|^ruby|^rake)"`
+
+    - name: Install Ruby ${{ matrix.ruby }}
+      uses: ruby/setup-ruby@v1.59.0
+      with:
+        ruby-version: ${{ matrix.ruby }}
+
+    - name: Cache RubyGems
+      uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-${{ matrix.ruby }}-gems-unit-${{ hashFiles(format('{0}.lock', matrix.gemfile || 'Gemfile')) }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.ruby }}-gems-unit-
+
+    - name: Install RubyGems
```

Comment:
Should this be handled by `RAILS_ENV=test ./script/rails-post-deploy` as per travis? Assuming that causes some issues?

Reply:
Initially I had the `ruby/setup-ruby` action install the gems which it can do along with saving and restoring the bundle cache - all as one step instead of three. I had to split this out to run `bundle config build.libv8 --with-system-v8`.

I guess we could use the `rails-post-deploy` script but I think I prefer having these as seporate workflow steps.

Reply:
I guess there's more chance of us missing things we add to `rails-post-deploy` in future by not doing so – e.g. `db:seed` is called in `rails-deploy-while-down`. I don't have a really strong preference right now though.

Reply:
Well we don't need the test DB seeded but I see your point if new actions are added in the future. Personally I think these deployment script are confusing and the GH action steps as they are simplier and easier to follow.

Lets give it a go as it is for now.